test/ingestor.tests.js

```diff
@@ -0,0 +1,61 @@
+const expect = require('chai').expect;
+const { filewalker, state } = require('../ingestor.js');
+
+describe('File Walker Ingestor', () => {
+  const testDirectory = './unzippedArchive';
+  let testState;
+
+  before(done => {
+    filewalker(testDirectory, (err, data) => {
+      if (err) {
+        throw err;
+      }
+      testState = state;
+      done();
+    });
+  });
+  after(done => {
+    testState;
+    done();
+  });
+
+  it('it should input a directory and output a state object that has a users key, a channels key, & a messages key', () => {
+    expect(testState).to.be.an('object');
+    expect(testState.users).to.be.an('array');
+    expect(testState.channels).to.be.an('array');
+    expect(testState.messages).to.be.an('array');
+  });
+
+  it('it should ouput a users array with a length of 2', () => {
```

Comment:
Can you tests the content within the testState, the same with the rest of your tests. I have a feeling these tests won't work.

Reply:
fixed!